moveit_experimental/moveit_jog_arm/src/low_pass_filter.cpp

```diff
@@ -76,8 +78,8 @@ double LowPassFilter::filter(double new_measurement)
   previous_measurements_[1] = previous_measurements_[0];
   previous_measurements_[0] = new_measurement;
 
-  double new_filtered_msrmt = (1. / (1. + filter_coeff_)) * (previous_measurements_[1] + previous_measurements_[0] -
-                                                             (-filter_coeff_ + 1.) * previous_filtered_measurement_);
+  double new_filtered_msrmt = scale_term_ * (previous_measurements_[1] + previous_measurements_[0] -
```

Comment:
I'm sure we can afford couple more letters here :wink:. I propose `new_filtered_measurement`.